src/app/services/chart.service.ts

```diff
@@ -2891,8 +2891,12 @@ export class ChartService {
       }
       // This updates this._outputDomains[key]['chart'][containerId].
       const value = me.calculateGDPValues(containerId, idx, numericValue, avgDoll, precision, oldValue);
+      const newChartId = chartId.slice(0, -2).replace('output-', '');
       console.log("debug", chartId, value);
+
       jQuery(`#${containerId} #${chartId} .text-number`).html(value);
+      // Hack for updating chart text on policy priority list
+      jQuery(`#${containerId} #${newChartId} .text-number`).html(value);
```

Comment:
Not sure why the Id's for these charts are so different, but this will target the id's seen on the policy priority charts.  ☹️